### PR TITLE
Ibp 3182 observation units brapi

### PR DESCRIPTION
**Pull request by vmaletta**
Please merge

### CODE REVIEW

File: src/main/java/org/generationcp/middleware/domain/search_request/ObservationUnitsSearchRequestDto.java [hunk 1]
```diff
@@ -0,0 +1,101 @@
+package org.generationcp.middleware.domain.search_request;
```
Comment (clarysabel) [inline src/main/java/org/generationcp/middleware/domain/search_request/ObservationUnitsSearchRequestDto.java]: we should have two packages? for v1 and v2?

File: src/main/java/org/generationcp/middleware/service/api/phenotype/PhenotypeSearchDTO.java [hunk 2]
```diff
@@ -1,5 +1,9 @@
+
+
```
Comment (clarysabel) [inline src/main/java/org/generationcp/middleware/service/api/phenotype/PhenotypeSearchDTO.java]: @vmaletta remove empty lines

File: src/main/java/org/generationcp/middleware/domain/search_request/ObservationUnitsSearchRequestDto.java (hunk 1 above)
Comment (clarysabel) [reply src/main/java/org/generationcp/middleware/domain/search_request/ObservationUnitsSearchRequestDto.java]: @vmaletta please reorganize the search requests in v1 and v2

Comment (vmaletta) [reply src/main/java/org/generationcp/middleware/domain/search_request/ObservationUnitsSearchRequestDto.java]: ok

File: src/main/java/org/generationcp/middleware/service/api/phenotype/PhenotypeSearchDTO.java (hunk 2 above)
Comment (vmaletta) [reply src/main/java/org/generationcp/middleware/service/api/phenotype/PhenotypeSearchDTO.java]: ok

**Pull request merged**